README.md

```diff
@@ -185,10 +183,13 @@ And of course this script must be run at least with profiling enabled and the au
 SPX_ENABLED=1 SPX_REPORT=full SPX_AUTO_START=0 my_script.php
 ```
 
+Automatic start can also be disabled for web requests via the `spx.http_profiling_auto_start` INI parameter or via the control panel.
+
+
 Side notes:
 - `spx_profiler_start()` and `spx_profiler_stop()` can safely be nested.
-- when automatic start is disabled, no signal handlers (i.e. on SIGINT/SIGTERM) are registered by SPX.
-- automatic start can only be disabled for CLI SAPI.
+- When profiling with the _full_ report type, `spx_profiler_stop()` returns the report key so that you will be able to store it somewhere, for instance among other information related to the profiled span. With the report key you can build the analysis screen URL which ends with this pattern `/?SPX_UI_URI=/report.html&key=<report key>`.  
```

Comment:
```suggestion
- when profiling with the _full_ report type, `spx_profiler_stop()` returns the report key so that you will be able to store it somewhere, for instance among other information related to the profiled span. With the report key you can build the analysis screen URL which ends with this pattern `/?SPX_UI_URI=/report.html&key=<report key>`.  
```